test/pylib/manager_client.py

```diff
@@ -289,7 +289,8 @@ def _create_server_add_data(self, replace_cfg: Optional[ReplaceConfig],
                                 property_file: Optional[dict[str, Any]],
                                 start: bool,
                                 seeds: Optional[List[IPAddress]],
-                                expected_error: Optional[str]) -> dict[str, Any]:
+                                expected_error: Optional[str],
+                                server_encryption: str) -> dict[str, Any]:
```

Comment:
could also use the type hint for `Optional[str]` (as we are checking if provided)